src/expansion_context.mli

```diff
@@ -2,9 +2,23 @@ module Base : sig
   (** Type for the location independent parts of the expansion context *)
   type t
 
-  (** Return the code path for the given context *)
+  (** Return the code path for the given context
+  In Driver, Deriving and Extension, the context is initialized so that the [file_path] component of
+  the [code_path] is determined from the first location found in the input AST. That means that:
+  - It's the empty string in empty structures or signatures
+  - It can be altered by earlier running PPX's
```

Comment:
Is that actually true? Don't we instantiate it only once at the beginning? I'm genuinely asking, I'm not sure exactly how it is initialized.

Reply:
Good catch! That's not true. I've removed that.